packages/launchpad/src/setup-actions.ts

```diff
@@ -123,6 +124,16 @@ export const setUpActionsForTasks = ( {
 						window.location.assign( `/subscribers/${ siteSlug }` );
 					};
 					break;
+				case 'site_launched':
+					action = async () => {
+						await wpcomRequest( {
+							path: `/sites/${ siteSlug }/launch`,
+							apiVersion: '1.1',
+							method: 'post',
+						} );
+						window.location.reload();
+					};
```

Comment:
@nelsonec87 and I discussed this on a call: it's not clear whether this is the right UX for site launch. We may want to drop the user into the general settings page, or we may want to launch the site and show a congratulatory banner in Customer Home, or something else.